src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -124,7 +124,7 @@ class AbstractIntegrationDetailedView<
   //Returns the string that is shown as the title of a tab
   getTabDiplay(tab: Tab): string {
     //default is return the tab
-    return tab;
+    return tab === 'information' ? 'overview' : 'configurations';
```

Comment:
@NisanthanNanthakumar If we are going to use `overview` instead of `information`, let's change it everywhere to be consistent. In other words, `type Tab =  'overview' | 'configurations'`;